dist/ramda.js

```diff
@@ -6354,6 +6385,7 @@
         multiply: multiply,
         nAry: nAry,
         negate: negate,
+        none: none,
```

Comment:
We're no longer updating __dist/ramda.js__ with each commit. Could you rebase your branch and undo the changes to this file?